### PR TITLE
Add Legacy Image Formats demo

### PR DESCRIPTION
**Pull request by loonybear**
Adding a demo for Legacy Image Formats

### CODE REVIEW

**Comment by ebidel** [comment]
Oh terrific. Thanks for adding this!

File: public/js/policies.json [hunk 1]
```diff
@@ -102,4 +102,22 @@
       "&lt;iframe allow=\"animations https:\/\/anim8.cdn.com\" src=\"...\"><\/iframe>"
     ]
   }
+}, {
+  "id": "legacy-image-formats",
+  "name": "Legacy Image Formats",
+  "url": "/demos/legacy-image-formats.html",
+  "what": "Restricts images to be one of certain modern formats (JPEG, PNG, GIF, WEBP, and SVG). Images with legacy formats will be rendered with inverted color.",
```
Comment (ebidel) [inline public/js/policies.json]: Restricts images to be one of the more modern formats (JPEG, ...

File: public/js/policies.json [hunk 2]
```diff
@@ -102,4 +102,22 @@
...
+  "why": "Ensures optimized performance with images.",
+  "usage": {
+    "off": "legacy-image-formats 'self'",
+    "on": "legacy-image-formats 'none'"
+  },
+  "examples": {
+    "header": [
+      "FeaturePolicy: legacy-image-formats 'none'"
```
Comment (ebidel) [inline public/js/policies.json]: Feature-Policy

**Comment by loonybear** [comment]
Done. Please take another look. Thanks

**Pull request merged**